src/components/sidebar/routePathView/routePathInfoTab/RoutePathInfoTab.tsx

```diff
@@ -1,54 +1,352 @@
+                                validationResult={
+                                    invalidPropertiesMap['destinationFi']
+                                }
+                            />
+                        </div>
+                        <div className={s.flexRow}>
+                            <InputContainer
+                                label='LÄHTÖPAIKKA RUOTSIKSI'
+                                disabled={isEditingDisabled}
+                                value={routePath.originSw}
+                                onChange={onChange('originSw')}
+                                validationResult={
+                                    invalidPropertiesMap['originSw']
+                                }
+                            />
+                            <InputContainer
+                                label='PÄÄTEPAIKKA RUOTSIKSI'
+                                disabled={isEditingDisabled}
+                                value={routePath.destinationSw}
+                                onChange={onChange('destinationSw')}
+                                validationResult={
+                                    invalidPropertiesMap['destinationSw']
+                                }
+                            />
+                        </div>
+                        <div className={s.flexRow}>
+                            <InputContainer
+                                label='LYHENNE SUOMEKSI'
+                                disabled={isEditingDisabled}
+                                value={routePath.routePathShortName}
+                                onChange={onChange('routePathShortName')}
+                                validationResult={
+                                    invalidPropertiesMap['routePathShortName']
+                                }
+                            />
+                            <InputContainer
+                                label='LYHENNE RUOTSIKSI'
+                                disabled={isEditingDisabled}
+                                value={routePath.routePathShortNameSw}
+                                onChange={onChange('routePathShortNameSw')}
+                                validationResult={
+                                    invalidPropertiesMap['routePathShortNameSw']
+                                }
+                            />
+                        </div>
+                        <div className={s.flexRow}>
+                            <InputContainer
+                                label='VOIM. AST'
+                                disabled={isUpdating}
+                                type='date'
+                                value={routePath.startTime}
+                                onChange={this.onChangeStartTime}
+                                validationResult={
+                                    invalidPropertiesMap['startTime']
+                                }
+                            />
```

Comment:
Hmm, maybe this inputContainer needs a new prop momentFormat?: string

then you could say:
momentFormat={'DD.MM.YYYY'}

and at inputContainer:
value={Moment(props.value).format(this.props.momentFormat ? momentFormat : 'DD.MM.YYYY HH:mm)}

(feel free to figure better variable name than momentFormat)

Reply:
https://github.com/HSLdevcom/jore-map-ui/issues/863